src/smc-webapp/smc_chat.cjsx

```diff
@@ -362,6 +440,37 @@ ChatLog = rclass
                 else
                     @props.actions.set_editing(@props.messages.get(date), false)
 
+    find_read_tos: (read_dates_by_user, message_dates) ->
+        read_indexes_to_user_ids = {}
+        [user_ids, read_dates] = [Object.keys(read_dates_by_user), underscore.values(read_dates_by_user)]
+        find_between = (read_date) ->
```

Comment:
find_between appears twice. just make it its own function and use it here. makes it shorter and more readable :-)